.github/workflows/pypi-package.yml

```diff
@@ -0,0 +1,22 @@
+name: Building & Publishing
+
+on:
+  push:
+    tags: ["v*"]
+
+permissions:
+  contents: read
```

Comment:
I reverted this back to only `read` permissions so that GitHub Actions does not have the ability to make commits, etc.

Reply:
I'm not seeing the `id-token: write` perm on any of the other ST workflows. Can we remove it?